plugins/services/src/js/service-configuration/ServiceNetworkingConfigSection.js

```diff
@@ -169,6 +171,33 @@ class ServiceNetworkingConfigSection extends ServiceConfigBaseSectionDisplay {
               });
             }
 
+            const containsServicePorts = portDefinitions.filter(function(
+              portDefinition
+            ) {
+              const result = portDefinition.servicePort
```

Comment:
So we need to detect if there's at least one `portDefinition` with a `servicePort` that is not `0`
will just `return portDefinition.servicePort != null && portDefinition.servicePort !== 0` work?